docs/src/snap/reference/certificates.md

```diff
@@ -0,0 +1,45 @@
+# Cluster Certificates
+
+## **CertificateAuthorities (CA)**
```

Comment:
Just curious, is there a particular reason for the stars here?

Reply:
not really, I think I just tried out if this makes it bold.